.circleci/config.yml

```diff
@@ -42,10 +41,22 @@ commands:
           name: Restore pnpm Package Cache
           keys:
             - node-deps-{{ arch }}-v3-{{ checksum "pnpm-lock.yaml" }}
-      - run:
-          name: Install pnpm package manager
-          command: |
-            npm install -g @pnpm/exe@8.3.1
+      - when:
+          condition:
+            equal: [<< parameters.os >>, linux]
+          steps:
+            - run:
+                name: Install pnpm package manager (linux)
+                command: |
+                  npm install --prefix=$HOME/.local -g @pnpm/exe@8.3.1
```

Comment:
Since we didn't install node in the $HOME folder, we can't install pnpm globally due to access rights on linux, so we need to specify target folder within our $HOME folder